social_network_api/src/main/java/com/danit/socialnetwork/MyDelegatedEncoder.java

```diff
@@ -0,0 +1,35 @@
+package com.danit.socialnetwork;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.crypto.password.DelegatingPasswordEncoder;
+import org.springframework.security.crypto.password.PasswordEncoder;
+
+import java.util.HashMap;
+
+@Configuration
+public class MyDelegatedEncoder {
```

Comment:
Do we really need this?

Reply:
I change this on BCryptPasswordEncoder()